pages/api/hello.ts

```diff
@@ -0,0 +1,13 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```

Comment:
Seems like maybe this was an initial test page. Is this hello route still actually used?

Reply:
I'll have to double check, I'm not sure off the top of my head